stytch/version.py

```diff
@@ -1 +1 @@
-__version__ = "5.13.0"
+__version__ = "5.13.1"
```

Comment:
This can be a minor version
```suggestion
__version__ = "5.14.0"
```